roles/ceph-osd/tasks/scenarios/non-collocated.yml

```diff
@@ -1,4 +1,82 @@
 ---
+- name: scenario II - OSDs round-robin their journals on dedicated devices
+  fail:
+    msg: "OSD scenario {{ osd_scenario }} is not non-collocated"
+  when: not non_collocated
+
+# ceph.service does not appear to be configured anymore.  Need to find a
+# way to ensure idempotency on the devices and dedicated_devices with 
+# respect to osd_provision boolean variable.
+
+# Each OSD device in the devices list has a corresponding journal device in
+# the dedicated_devices list.  Use round-robin to fill up the remaining
+# entires in the dedicated_devices to match the entries in the devices list.
+#
+# For example, assuming the devices list has these entries on a system
+# devices: [ /dev/sdt, /dev/sdu, /dev/sdv, /dev/sdw, /dev/sdx ]
+# On this system, it has two SSD disks
+# dedicated_devices: [ /dev/sdb, /dev/sdc ]
+#
+# In this use case, we need to add three more entries to dedicated_devices:
+# dedicated_devices: [ /dev/sdb, /dev/sdc, /dev/sdb, /dev/sdc, /dev/sdb ]
+#
+# Each distinct device in dedicated_devices can only be used up to the ratio
+# set in the osd_data_journal_ratio tunable.
+- name: Initialze temporary variable for populating dedicated_devices list
```

Comment:
please remove capital letters, also use this convention:

`name: set_fact journal_cnt <description>`